routing/router.go

```diff
@@ -338,3 +343,36 @@ func serveIndexFile(w http.ResponseWriter) {
 		log.Errorf("HTTP I/O error [%v]", err.Error())
 	}
 }
+
+// Things to help build the logger handler chain (see https://pkg.go.dev/github.com/rs/zerolog/hlog#example-package-Handler)
+// fake alice to avoid dep
+type alice struct {
+	m []func(http.Handler) http.Handler
+}
+
+func (a alice) append(m func(http.Handler) http.Handler) alice {
+	a.m = append(a.m, m)
+	return a
+}
+func (a alice) then(h http.Handler) http.Handler {
+	for i := range a.m {
+		h = a.m[len(a.m)-1-i](h)
+	}
+	return h
+}
+func buildHttpHandlerLogger(route Route, handlerFunction http.Handler) http.Handler {
+	c := alice{}
+	c = c.append(hlog.NewHandler(log.WithGroup(route.Name).Z))
+	c = c.append(hlog.HostHandler("host", true))
+	c = c.append(hlog.HTTPVersionHandler("http_version"))
+	c = c.append(hlog.MethodHandler("method"))
+	c = c.append(hlog.ProtoHandler("proto"))
+	c = c.append(hlog.RefererHandler("referer"))
+	c = c.append(hlog.RemoteAddrHandler("remote_addr"))
+	c = c.append(hlog.RemoteIPHandler("remote_ip"))
+	c = c.append(hlog.RequestIDHandler("request_id", "Request-Id"))
```

Comment:
IMO we should only add `Request-Id` for info level logs. Some of them, like `URL` may have sensitive info in them if it's an auth redirect.

Also -1 on snake case. Unless this is a convention for logging nothing else in the app is logged this way. Seems like it should either be `RequestID` or RequestId`.

Reply:
I removed them all but request ID and renamed it.